ts/private/ts_validate_options.bzl

```diff
@@ -51,7 +51,12 @@ To disable this check, set the validate attribute to False:
         incremental = ctx.attr.incremental,
         ts_build_info_file = ctx.attr.ts_build_info_file,
     )
-    arguments.add_all([ctx.file.tsconfig.short_path, marker.short_path, ctx.attr.target, json.encode(config)])
+    arguments.add_all([
+        ctx.file.tsconfig.path.removeprefix("%s/" % ctx.bin_dir.path),
```

Comment:
Can use `to_output_relative_path` for these two as well

Reply:
Done. Same issue here where `tsconfig.json` was being copied to bin after using it's path. Hoisted the copy to bin earlier in the rule. `marker` didn't have this problem since it is created internally in the rule, so it's always in the bin directory.